experimental/desktop_photo_search/macos/Runner.xcodeproj/project.pbxproj

```diff
@@ -431,6 +430,7 @@
 				CODE_SIGN_ENTITLEMENTS = Runner/DebugProfile.entitlements;
 				CODE_SIGN_STYLE = Automatic;
 				COMBINE_HIDPI_IMAGES = YES;
+				DEVELOPMENT_TEAM = S8QB4VV633;
```

Comment:
Nip these out unless you need them for something.

Reply:
done